components/ProgressBars/index.js

```diff
@@ -0,0 +1,76 @@
+import {
+  CircularProgressbar,
+  CircularProgressbarWithChildren,
+  buildStyles,
+} from "react-circular-progressbar";
+import { ProgressbarContainer } from "./ProgressBar.styled";
+import { useState, useEffect } from "react";
+import { CarbWrapper, ProteinWrapper, FatWrapper } from "./ProgressBar.styled";
+import {
+  kcalStyles,
+  carbStyles,
+  proteinStyles,
+  fatStyles,
+} from "./ProgressBar.styled";
+
+export default function ProgressBars({
+  todaysKcal,
+  todaysCarbs,
+  todaysProtein,
+  todaysFat,
+  goal,
+}) {
+  const [kcalProgress, setKcalProgress] = useState(todaysKcal);
+  const [carbsProgress, setCarbsProgress] = useState(todaysCarbs);
+  const [proteinProgress, setProteinProgress] = useState(todaysProtein);
+  const [fatProgress, setFatProgress] = useState(todaysFat);
+
+  useEffect(() => {
+    setKcalProgress(todaysKcal);
+    setCarbsProgress(todaysCarbs);
+    setProteinProgress(todaysProtein);
+    setFatProgress(todaysFat);
+  }, [todaysKcal, todaysCarbs, todaysProtein, todaysFat]);
+
+  return (
+    <ProgressbarContainer>
+      <CircularProgressbarWithChildren
+        value={kcalProgress}
+        maxValue={goal.kcal}
+        circleRatio={0.5}
+        strokeWidth={10}
+        styles={buildStyles(kcalStyles)}
```

Comment:
Here you can spread the nutrientStyles object and add the individual pathColor to your styles. This also would be true for the macro nutrients.

```suggestion
        styles={
         ...buildStyles(nutrientStyles),
         pathColor: "var(--color-fg-kcal)"
         }
```